lib/pages/app_grid/app_grid_dashboard_controller.dart

```diff
@@ -50,14 +52,33 @@ class AppGridDashboardController extends State<AppGridDashboard> {
       Logs().d(
         'AppGridDashboardController::handleGetAppGridDashboardSuccess(): $linagoraApps',
       );
-      linagoraApplications.value = linagoraApps;
+      linagoraApplications.value = _getActiveLinagoraApps(linagoraApps);
     } on FlutterError catch (e) {
       Logs().e(
         'AppGridDashboardController::handleShowAppDashboard(): $e',
       );
     }
   }
 
+  LinagoraApplications _getActiveLinagoraApps(
+    LinagoraApplications linagoraApps,
+  ) {
+    final Map<String, bool> mapActiveApps = {
+      'Twake Chat': false,
+      'Contacts': false,
+      'Calendar': false,
+      'Teleskop': false,
+      'Twake Mail': true,
+      'Twake Drive': true,
+    };
```

Comment:
i think you can make a contant, and may be enum is better

Reply:
Updated